### ISSUE TITLE
Shadow Basketball (2005)

### ISSUE THREAD

**Issue by Difegue** [Difegue/LCDonald#11]
![Picture of the game to simulate](http://info.sonicretro.org/images/thumb/9/9b/Shadow_Basketball_Game.jpg/835px-Shadow_Basketball_Game.jpg)  

- [x] Game owned 
- [x] Manual owned and scanned
- [x] Photos taken (front, back, cover open or closed if there's one)
- [x] LCD elements digitized and vectorized to SVG
- [x] mamelayout file written
- [x] Simulator logic written

**Comment by gaurang7goel** [comment]
Hi, @Difegue  , I found this on the internet. Don't know if it is useful or not.

![Shadow_Basketball_Manual_cleanup](https://user-images.githubusercontent.com/25103902/221481453-60e008b5-d1e1-4de2-8185-e35980ee92bf.jpg)

**Comment by Georgedude** [comment]
Are you still updating artwork? I've checked my old memory boxes while looking for the physical games and found some English manuals for Shadow Basketball, Sonic Skateboard, Cream Flower Catch (yellow version), Amy and Rouge Tennis, Tails Football and Knuckles Baseball

If useful I'm happy to scan them over, I've also got some other bits including the box itself and other leaflets from the time (UK)

**Comment by Difegue** [comment]
Yup, still am! Although this is quite niche so I haven't really had much outside assistance to get the missing manuals.  

Any high-res scans would be much appreciated!

**Comment by Georgedude** [comment]
Sorry for the wait, I had to wait until I was at my relative's house to use their scanner :P Hopefully these can be of some use, though if there's any issues I'm happy to rescan :) I've also added a couple others here that I found (a leaflet and instructions for one of the Sonic disc spinners) just for fun and so they can exist somewhere on the internet

[AMY_ROUGE_TENNIS.pdf](https://github.com/user-attachments/files/18011181/AMY_ROUGE_TENNIS.pdf)
[CREAM_FLOWER_CATCH.pdf](https://github.com/user-attachments/files/18011182/CREAM_FLOWER_CATCH.pdf)
[FRONT_ELECTRONIC.pdf](https://github.com/user-attachments/files/18011183/FRONT_ELECTRONIC.pdf)
[KNUCKLES_BASEBALL.pdf](https://github.com/user-attachments/files/18011184/KNUCKLES_BASEBALL.pdf)
[SHADOW_BASKETBALL.pdf](https://github.com/user-attachments/files/18011185/SHADOW_BASKETBALL.pdf)
[SONIC_SKATEBOARD.pdf](https://github.com/user-attachments/files/18011186/SONIC_SKATEBOARD.pdf)
[SONIC_SPINNER_BACK.pdf](https://github.com/user-attachments/files/18011187/SONIC_SPINNER_BACK.pdf)
[SONIC_SPINNER_FRONT.pdf](https://github.com/user-attachments/files/18011188/SONIC_SPINNER_FRONT.pdf)
[SPAIN_PROMO_BACK.pdf](https://github.com/user-attachments/files/18011189/SPAIN_PROMO_BACK.pdf)
[SPAIN_PROMO_FRONT.pdf](https://github.com/user-attachments/files/18011190/SPAIN_PROMO_FRONT.pdf)
[TAILS_FOOTBALL.pdf](https://github.com/user-attachments/files/18011191/TAILS_FOOTBALL.pdf)

**Comment by Difegue** [comment]
Thank you very much!  
These are great already and I'll def use them - If possible it would be nice to check if you can crank up the DPI on your scanner so it makes larger scans.  

Reason I'm asking is that the manuals have more detailed drawings of the LCD portions, so I tend to try using those in the games as well if the scans are of good enough quality - See #6 for details.

**Comment by Georgedude** [comment]
Good shout, I haven’t got much experience with scanners so I’ll follow your advice to get the best picture quality possible, I’ll give them a rescan when I’m next able :)
